src/instructlab/taxonomy/diff.py

```diff
@@ -82,12 +82,14 @@ def diff(ctx, taxonomy_path, taxonomy_base, yaml_rules, quiet):
     # validate new or changed taxonomy files
     try:
         validate_taxonomy(taxonomy_path, taxonomy_base, yaml_rules)
-    except (TaxonomyReadingException, yaml.YAMLError) as exc:
+    except (TaxonomyReadingException, GitError, yaml.YAMLError) as exc:
         if not quiet:
             click.secho(
                 f"Reading taxonomy failed with the following error: {exc}",
                 fg="red",
             )
+        else:
```

Comment:
This seems wrong. In the quiet case, you are now very much not quiet by logging an error.

Reply:
+1. It seems like the quiet case is meant to be ensure this command returns no output at all. The changes in this commit make that not the case anymore.

Reply:
This was changed to log.debug which is probably fine for quiet mode since most users will not run with `-v`.